scripts/update_config.rb

```diff
@@ -0,0 +1,40 @@
+# I'm just pasting this into a console. hackity-hack-hack
+
+BASE_URL = 'https://api.heroku.com'
+sandwich = Sandwich.last
+
+if sandwich.access_token_expires_at < (Time.now.utc - 90)
+  AccessTokenRefresher.new(sandwich_id: sandwich_id).run
+end
+
+sandwich.reload
+
+heroku_uuid = sandwich.heroku_uuid
+access_token = sandwich.access_token
+
+response = Excon.new(BASE_URL).patch(
+  path: "/addons/#{heroku_uuid}/config",
+  headers: {
+    'Accept' => 'application/vnd.heroku+json; version=3',
+    'Authorization' => "Bearer #{access_token}",
+    'Content-Type' => 'application/json',
+  },
+  body: JSON.dump(
+    config: [
+      {
+        name: "DJCPWICH_SUDO_SANDWICH_COMMAND",
```

Comment:
I thought that you had to prepend all config vars with the name of your add-on? eg: `SUDO_SANDWICH_XXX`

Reply:
I don't want this merged, I'm just showing you what I did to reproduce the issue.